### PR TITLE
Add Div A field option to Simulator

### PR DESCRIPTION
**Pull request by akhilveeraghanta**
### Description
Add Div A field size + 11 robots to our custom sim and update example play to scale to 11 robots

### Testing Done
Manual testing

### Resolved Issues
partially implements #2009 (doesn't touch the simulated tests)

### Length Justification
N/A

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/software/ai/hl/stp/play/example_play.cpp [hunk 1]
```diff
@@ -53,12 +58,28 @@ void ExamplePlay::getNextTactics(TacticCoroutine::push_type &yield, const World
         move_tactic_6->updateControlParams(
             world.ball().position() + Vector::createFromAngle(angle_between_robots * 6),
             (angle_between_robots * 6) + Angle::half(), 0);
+        move_tactic_7->updateControlParams(
+            world.ball().position() + Vector::createFromAngle(angle_between_robots * 7),
```
Comment (jonathanlew) [inline src/software/ai/hl/stp/play/example_play.cpp]: can we use a for loop here?

File: src/software/simulation/standalone_simulator.cpp [hunk 2]
```diff
@@ -160,6 +160,57 @@ void StandaloneSimulator::setupInitialSimulationState()
...
+    RobotState blue_robot_state9(Point(2, -0.5), Vector(0, 0), Angle::half(),
+                                 AngularVelocity::zero());
+    RobotState blue_robot_state10(Point(2, -1.5), Vector(0, 0), Angle::half(),
+                                  AngularVelocity::zero());
+    RobotState blue_robot_state11(Point(2, -2.5), Vector(0, 0), Angle::half(),
+                                  AngularVelocity::zero());
+
+    std::vector<RobotStateWithId> blue_robot_states = {
+        RobotStateWithId{.id = 6, .robot_state = blue_robot_state6},
+        RobotStateWithId{.id = 7, .robot_state = blue_robot_state7},
```
Comment (jonathanlew) [inline src/software/simulation/standalone_simulator.cpp]: can this be refactored into a for loop

File: src/software/simulation/standalone_simulator.h [hunk 3]
```diff
@@ -47,7 +48,8 @@ class StandaloneSimulator
     /**
      * Adds robots to predefined locations on the field
      */
-    void setupInitialSimulationState();
+    void setupInitialSimulationStateDivA();
+    void setupInitialSimulationStateDivB();
```
Comment (jonathanlew) [inline src/software/simulation/standalone_simulator.h]: isn't this more about setting up 11 robots?

Comment (akhilveeraghanta) [reply src/software/simulation/standalone_simulator.h]: Yeah that's basically everything this does

File: src/software/ai/hl/stp/play/example_play.cpp [hunk 4]
```diff
@@ -20,12 +20,13 @@ bool ExamplePlay::invariantHolds(const World &world) const
...
 {
     // Create MoveTactics that will loop forever
-    auto move_tactic_1 = std::make_shared<MoveTactic>(true);
-    auto move_tactic_2 = std::make_shared<MoveTactic>(true);
-    auto move_tactic_3 = std::make_shared<MoveTactic>(true);
-    auto move_tactic_4 = std::make_shared<MoveTactic>(true);
-    auto move_tactic_5 = std::make_shared<MoveTactic>(true);
-    auto move_tactic_6 = std::make_shared<MoveTactic>(true);
+    std::vector<std::shared_ptr<MoveTactic>> move_tactics = {
+        std::make_shared<MoveTactic>(true), std::make_shared<MoveTactic>(true),
```
Comment (jonathanlew) [inline src/software/ai/hl/stp/play/example_play.cpp]: a fill contructor should work here http://www.cplusplus.com/reference/vector/vector/vector/

File: src/software/simulation/standalone_simulator.cpp [hunk 5]
```diff
@@ -113,50 +113,62 @@ void StandaloneSimulator::setupNetworking(int blue_team_channel, int yellow_team
...
-        RobotStateWithId{.id = 0, .robot_state = blue_robot_state1},
-        RobotStateWithId{.id = 1, .robot_state = blue_robot_state2},
-        RobotStateWithId{.id = 2, .robot_state = blue_robot_state3},
-        RobotStateWithId{.id = 3, .robot_state = blue_robot_state4},
-        RobotStateWithId{.id = 4, .robot_state = blue_robot_state5},
-        RobotStateWithId{.id = 5, .robot_state = blue_robot_state6},
-    };
+void StandaloneSimulator::setupInitialSimulationStateDivB()
+{
+    std::vector<RobotStateWithId> blue_robot_states;
```
Comment (jonathanlew) [inline src/software/simulation/standalone_simulator.cpp]: what if we had a function that took the number of robots and made a reasonably spaced configuration?

File: src/software/ai/hl/stp/play/example_play.cpp (hunk 4 above)
Comment (akhilveeraghanta) [reply src/software/ai/hl/stp/play/example_play.cpp]: very neat, didn't know this was a thing, updated

File: src/software/simulation/standalone_simulator.cpp (hunk 5 above)
Comment (akhilveeraghanta) [reply src/software/simulation/standalone_simulator.cpp]: yeah thats probably cleaner, updated

**Review by jonathanlew — DISMISSED** [review]
LGTM pending CI

File: src/software/ai/hl/stp/play/example_play.cpp (hunk 4 above)
Comment (akhilveeraghanta) [reply src/software/ai/hl/stp/play/example_play.cpp]: this doesn't work because it fills it with the same value

switched to `std::generate`

**Review by jonathanlew — APPROVED** [review]
Nice work! :eu:

**Review by itsarune — APPROVED** [review]
nothing to add from me

**Pull request merged**